front/app/components/Form/Components/Layouts/PageControlButtons.tsx

```diff
@@ -35,79 +34,61 @@ const PageControlButtons = ({
   showSubmit,
   dataCyValue,
 }: Props) => {
-  const modalPortalElement = document.getElementById('modal-portal');
   const theme = useTheme();
   const isSmallerThanPhone = useBreakpoint('phone');
 
-  return modalPortalElement
-    ? createPortal(
-        <Box
-          display="flex"
-          flexDirection="row"
-          justifyContent="center"
-          w="100%"
-          zIndex="1010"
-          position="fixed"
-          borderRadius="2px"
-        >
-          <Box
-            position="fixed"
-            bottom={isSmallerThanPhone ? '0px' : '40px'}
-            maxWidth="700px"
-            width="100%"
-            display="flex"
-            justifyContent="space-between"
-            alignItems="center"
-            bgColor={colors.white}
-            borderTop={`1px solid ${colors.borderLight}`}
-            padding={isSmallerThanPhone ? '16px' : '16px 24px'}
```

Comment:
The amount of CSS I've been able to delete that doesn't do anything boggles my mind. We should be much more careful when adding CSS and explain any non-obvious styles with comments because this was hard to work with.